modules/vba_documents/spec/mailers/unsuccessful_report_mailer_spec.rb

```diff
@@ -33,31 +33,59 @@
   end
 
   describe '#build' do
-    subject do
-      described_class.build(totals, [error_upload], [uploaded_upload], 7.days.ago, Time.zone.now).deliver_now
+    module MailerHelper
+      class << self
+        attr_reader :env_hash
+      end
+
+      @env_hash = {
+        staging: 'https://dsva-vetsgov-staging-benefits-documents.s3.us-gov-west-1.amazonaws.com/',
+        dev: 'https://dsva-vetsgov-dev-benefits-documents.s3.us-gov-west-1.amazonaws.com/',
+        sandbox: 'https://dsva-vagov-sandbox-benefits-documents.s3.us-gov-west-1.amazonaws.com/',
+        prod: 'https://dsva-vetsgov-prod-benefits-documents.s3.us-gov-west-1.amazonaws.com/'
+      }
```

Comment:
Is it possible to make the test not depend on files from AWS? This might be regular in tests vets-api, but I'm not familiar.

Reply:
Not that I am aware of.  The dependency is only during testing and I am not sure I see any difference between this and using VCR recordings.